spec/presenters/step_by_step_presenter_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+require "gds_api/test_helpers/link_checker_api"
+
+RSpec.describe StepByStepPresenter do
+  include GdsApi::TestHelpers::LinkCheckerApi
+  include AuthenticationControllerHelpers
+  include GovukContentSchemaTestHelpers
+  include TimeOptionsHelper
+
+  before do
+    allow(Services.publishing_api).to receive(:lookup_content_id)
+    stub_user.name = "Firstname Lastname"
+  end
+
+  describe "#summary_list" do
+    let(:step_nav) { create(:step_by_step_page_with_steps) }
+    let(:default_summary) {
+      [
+        {
+          field: "Status",
+          value: "Draft"
+        },
+        {
+          field: "Last saved",
+          value: format_full_date_and_time(Time.zone.now)
+        },
+        {
+          field: "Created",
+          value: format_full_date_and_time(Time.zone.now)
+        }
+      ]
+    }
+
+    subject { described_class.new(step_nav).summary_list }
+
+    it "presents a list of key/value metadata" do
+      expect(subject).to eq(default_summary)
+    end
+
+    context "assigned_to is set" do
+      let(:step_nav) { create(:step_by_step_page_with_navigation_rules, assigned_to: stub_user.name) }
+
+      it "shows who made the most recent change" do
+        expect(subject[1]).to eq(
```

Comment:
`subject[1]` could start failing if the order of the information changes. Perhaps a `select` could be used to find the hash with a `field` that equals "Last saved". 

See: https://stackoverflow.com/questions/2244915/how-do-i-search-within-an-array-of-hashes-by-hash-values-in-ruby